plugins/modules/cloud/docker/docker_image.py

```diff
@@ -737,11 +743,14 @@ def build_image(self):
         if self.target:
             params['target'] = self.target
 
+        build_output = []
         for line in self.client.build(**params):
             # line = json.loads(line)
             self.log(line, pretty_print=True)
-            if "stream" in line:
-                build_output.append(line["stream"])
+            if "stream" in line or "status" in line:
+                build_line = line.get("stream") or line.get("status")
+                build_output.append(build_line.strip())
```

Comment:
```suggestion
                build_output.append(build_line)
```
I don't think we should modify the output.

Reply:
All right, removed it